tools.html

```diff
@@ -155,13 +156,13 @@ <h2 class="text-center my-16 text-3xl font-bold tracking-tight text-dark md:text
       %}
 
     {% include card_tool.html
-      title="Microkit VMM"
+      title="libvmm"
```

Comment:
The CAmkES VMM lib is also called `libvmm`. How about "Microkit VMM Library" (and CAmkES VMM Library)?

Reply:
I thought it was `camkes-vm` and `camkes-vm-examples`?

Reply:
We can keep it as Microkit VMM library if it's too confusing, it's just that libvmm is intended to be OS agnostic (although right now isn't).

Reply:
`camkes-vm` is just the name of the repo. It's also just a library in the end.

Reply:
It's not a library, it has a main.c and a main function I believe. It's a bit different to what libvmm provides.

Reply:
Alright, I've merged the current version (with library) so that we have the fixes in before announcements go out. Happy to tweak more later.

Reply:
No worries. Thanks.